### ISSUE TITLE
Раздел Блог "URL к папке с изображениями *"

### ISSUE THREAD

**Issue by matrixsx** [Globulopolis/ucoz_converter#2]
"URL к папке с изображениями *" не меняются значения после переноса ссылка на картинку так и остается прежняя

**Comment by Globulopolis** [comment]
Эта опция влияет только на изображения в теге <!--IMG...
Это только вложения.
Если нужно поменять другие ссылки, есть `Список замены ссылок` на первой вкладке.

**Comment by matrixsx** [comment]
Именно
img src="1/blogfoto/samsung_toner.jpg" alt="Картридж Samsung" width="225" height="225" align="left"  моя картинка в теге но переностится в  статью она так же как и скопировалась из статьи, может я не так понимаю функционал данного поля?

**Comment by Globulopolis** [comment]
Автоматически заменяется только изображения вида ```<!--IMG3--><img alt="" style="margin:0;padding:0;border:0;" src="http://сайт/_bl/.../....jpg" align="" /><!--IMG3-->```

Если вам нужно заменить адрес в ```<img ... src="http://сайт/_bl/0/43839339.jpg" ... />``` укажите старый и новый адрес в `Список замены ссылок`.

Также необходимо указать старый адрес сайта в поле `Старый адрес сайта`.

**Comment by matrixsx** [comment]
Смотрите: 
есть статья на ucoz с картинками например тег такой img src="1/blogfoto/samsung_toner.jpg" alt="Картридж Samsung" width="225" height="225" align="left"
когда статья переносится конвертером в joomla она переносится так же img src="1/blogfoto/samsung_toner.jpg" но картинки уже не доступны по этому адресу т.к нет такого пути новом хостинге. я думал что все вложения статьи перенесутся в папку которая указана в поле "URL к папке с изображениями *"

**Comment by Globulopolis** [comment]
Он заменяет, но только у вложений. Если вы руками прописывали изображения - замены не будет.
Используйте `Список замены ссылок` и после импорта, перенесите изображения вручную по новому пути.

**Comment by matrixsx** [comment]
А можно реализовать перенос картинок которые не в вложении а в статье вставлены так же в папку? пропарсил статью и картинки закинул в папку определенную

**Comment by Globulopolis** [comment]
Думаю что можно.

**Comment by Globulopolis** [comment]
@matrixsx пример папки приведите или ссылки на картинку,

**Comment by matrixsx** [comment]
4cartridges. net/blog/pervye_v_mire_printery_samsung_na_android/2014-09-15-35
к примеру эта статья

**Comment by Globulopolis** [comment]
@matrixsx есть одна проблема. Ссылка на картинку в блогах Ucoz ведет к папке `_bl/цифра/`, а картинка из вашего примера ведет в папку просто в корне сайта.
Из вашего примера картинка по идее должна быть по пути `_bl/2175/картинка.жпг`. С учетом кол-ва вариантов указания пути(от корня, с адресом или без, условный или вообще через //) имхо проще всё же руками перенести изображения.

**Comment by matrixsx** [comment]
согласен, у меня нет картинок почему то в _bl/ * все картинки раскиданы есть в корне есть в папках
А когда заработает   
 Converter for loads
 Converter for news
 Converter for publications?

**Comment by Globulopolis** [comment]
> А когда заработает
> Converter for loads
> Converter for news
> Converter for publications?

Как только так сразу.

**Comment by matrixsx** [comment]
файлы скопировал в одну папку /imports/blog/ картинки.jpg
но при миграции получается ерунда какая то 
вот так в статье прописываются картинки
сайт.ру/images/blog/2175images/blog/Xerox_Phaser_3052.jpg
уже как только не пытался
картинки на ucoz лежат и в папке /2175 и в /1 и в /245/ можно как то указать чтобы не важно где лежат картинки менялось все на /imports/blog/

**Comment by Globulopolis** [comment]
В списке `Список замены ссылок` есть что-то?

**Comment by matrixsx** [comment]
Старый URL | Новый URL | 
/                       | images/blog
в первый стобец пробовал /* ничего не выходит , получается только вот так:

Старый URL | Новый URL | 
/2175                      | images/blog
тогда все отлично работает у тех статей у которых картинки лежат /2175

**Comment by Globulopolis** [comment]
`Старый URL | Новый URL` что ввели то и будет искать и заменять. Т.е. точное соответствие и никаких регулярных выражений и подстановочных символов. Если не сложно выложите на облако mail,ru бэкап блога. Посмотрю что можно сделать.

**Comment by matrixsx** [comment]
можно вашу почту или контакты?

**Comment by Globulopolis** [comment]
https://киноархив.com/en/support/contacts.html

**Comment by matrixsx** [comment]
> [https://киноархив.com/en/support/contacts.html](https://%D0%BA%D0%B8%D0%BD%D0%BE%D0%B0%D1%80%D1%85%D0%B8%D0%B2.com/en/support/contacts.html)

отправил

**Comment by Globulopolis** [comment]
Забрал. Можете убирать с Яши. Будем посмотреть что там да как.

**Comment by matrixsx** [comment]
Получилось что нибудь сделать ?

**Comment by Globulopolis** [comment]
Работаю над этим.

**Comment by matrixsx** [comment]
> Работаю над этим.

как успехи?
